assets/state-dra-driver/0200_clusterrole.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: nvidia-dra-driver
+rules:
+# TODO: restrict RBAC for DRA driver
```

Comment:
Has this been done upstream yet?

Reply:
Not yet, see https://github.com/NVIDIA/k8s-dra-driver/blob/main/deployments/helm/k8s-dra-driver/templates/clusterrole.yaml

Issue tracker: https://github.com/NVIDIA/cloud-native-team/issues/134

Reply:
I see @guptaNswati  is working on restricting the RBAC upstream: https://github.com/NVIDIA/k8s-dra-driver/pull/219. I will pull in those changes once they are finalized.